.eslintrc.yml

```diff
@@ -32,7 +32,12 @@ overrides:
       ecmaVersion: 6
```

Comment:
Can't we update this (ES6 === ECMAScript 2015) to ECMAScript 2018 as well?
And use `async`/`await` especially in our "lib/cli/**/*.js" files?

Reply:
I would prefer that, but I didn't want to change anything that major. Which is why we get my shenanigans around that.

If you say yes, I can make the change. It will probably also mean transpiling async/await in the babel config.

Reply:
Our main problem is IE11 (to be dropped in 2025 😁), that's why we have to use ES5.
Our "lib/cli/**/*.js" files have been overwritten to ES6 for months. This means:
- our browser part does not use those files (seems very plausible)
- since we dropped Node v6, we can update from ES6 to ES2018 (ES5 remains default)
- we should exclude those "lib/cli/**/*.js" files while browserify'ing

So from my side you get a go. Let's wait for some days, if there is no objection, then my will be done.

Reply:
updated from ES6 to ES2017 in separate PR.